src/CRM/CivixBundle/Resources/views/Code/module.civix.php.php

```diff
@@ -166,7 +166,7 @@ function _<?php echo $mainFile ?>_civix_civicrm_uninstall() {
  *
  * @link https://docs.civicrm.org/dev/en/latest/hooks/hook_civicrm_enable
  */
-function _<?php echo $mainFile ?>_civix_civicrm_enable() {
+function _<?php echo $mainFile ?>_civix_civicrm_enable(): void {
```

Comment:
Long history of no return

![image](https://user-images.githubusercontent.com/336308/196287264-2e0e4c47-f578-4ef9-b367-b3c872de5597.png)